machine_management/control_plane_machine_management/cpmso-configuration.adoc

```diff
@@ -33,6 +33,8 @@ The `<platform_provider_spec>` and `<platform_failure_domains>` sections of the
 
 * xref:../../machine_management/control_plane_machine_management/cpmso-configuration.adoc#cpmso-sample-yaml-vsphere_cpmso-configuration[Sample YAML snippets for configuring VMware vSphere clusters]
 
+* xref:h../../machine_management/control_plane_machine_management/cpmso-configuration#cpmso-sample-yaml-openstack_cpmso-configuration[Sample YAML snippets for configuring {rh-openstack} clusters]
```

Comment:
typo. Would also suggest using the full name (`rh-openstack-first`) here to match the others
```suggestion
* xref:../../machine_management/control_plane_machine_management/cpmso-configuration#cpmso-sample-yaml-openstack_cpmso-configuration[Sample YAML snippets for configuring {rh-openstack} clusters]
```

Reply:
Ack.